css-positioning/absolute.css

```diff
@@ -0,0 +1,46 @@
+.meme {
+  width: 300px;
+  height: 300px;
+  background-image: url('images/one-does-not-simply.jpg');
+  background-size: cover;
+
+  /* your code here */
+  position: relative;
+}
+
+.meme > img {
+  width: 100%;
+}
+
+.emoji {
+  font-size: 3rem;
+
+  /* your code here */
+  position: absolute;
+  top: 17px;
+  right: 17px;
+}
+
+.joke {
+  margin-bottom: 1rem;
+  font-size: 2rem;
+  font-family: Anton, sans-serif;
+  text-align: center;
+  color: white;
+  -webkit-text-stroke: 1px black;
+  text-transform: uppercase;
+
+  /* your code here */
+  position: absolute;
+  bottom: 0;
+}
+
+.overlay {
+  background: linear-gradient(transparent, rgba(0, 0, 0, 0.75));
+
+  /* your code here */
+
+  position: absolute;
+  bottom: 0;
+  top: 0;
+}
```

Comment:
This overlay isn't showing up over the `.meme` image. While it has the correct `height` starting from 0 top and 0 bottom, where is it's `width` spanning the left and right of the parent?